app/helpers/application_helper.rb

```diff
@@ -129,6 +129,13 @@ def present_from_to(from, to)
     end
   end
 
+  def govuk_date_range(from, to)
+    from = from.to_date if from.respond_to?(:to_date)
+    to = to.to_date if to.respond_to?(:to_date)
```

Comment:
Is it possible that  `from`  responds to `to_date` but not `to`? 🤔 
(or vice-versa)

is it acceptable?

if not a guard clause might be a better choice
```
return unless from.respond_to?(:to_date) && and to.respond_to?(:to_date)

"#{from.to_formatted_s(:long)} to #{to.to_formatted_s(:long)}"

```